packages/jest-cli/src/TestRunner.js

```diff
@@ -16,9 +16,14 @@ import type {
 import type {GlobalConfig, ReporterConfig} from 'types/Config';
 import type {Context} from 'types/Context';
 import type {Reporter, Test} from 'types/TestRunner';
-import type {PathPattern} from './SearchSource';
+import type {PathPatternObject} from './SearchSource';
 
 import {formatExecError} from 'jest-message-util';
+import {
+  addResult,
+  buildFailureTestResult,
+  makeEmptyTestResult,
+} from './test_result_utils';
```

Comment:
I was referring just to rename `makeEmptyTestResult` -> `makeEmptyAggregatedResult` if that makes sense